### ISSUE TITLE
Feature Request: A command to show project of current note

### ISSUE THREAD

**Issue by joellti** [kevboh/longform#145]
**Describe the solution you'd like**
Add a command which shows Longform pane with correct project title set and the scene matching the current active note highlighted.

**Describe alternatives you've considered**
Multiple steps are required to achieve the same.
- Get the project title by revealing the current active note in File Explorer pane
- Switch to Longform pane and select the project title from dropdown

**Comment by kevboh** [comment]
Yeah, right now this takes two commands: "Open Current Note's Project" followed by "Open Longform Pane". The former should do both. Thanks.

**Comment by joellti** [comment]
Thanks @kevboh for your input. I was able to use Commander plugin's Macro function to combine these two commands into a single command and I think it is a good workaround for now.
